tests/test_keras_integration.py

```diff
@@ -274,6 +283,7 @@ def test_push_to_hub(self):
         REPO_NAME = repo_name("PUSH_TO_HUB")
         model = self.model_init()
         model.build((None, 2))
+        self.model_fit(model)
```

Comment:
This adds in the assumption that the model was fit, eh? I think there should be a test that shows you can load a trained model from file + can push that as well without errors.

Reply:
If you require to be fit, btw, you don't need to build anymore, right? Its a meaningless step then if I remember correctly. I could be wrong!!

Reply:
I don't require to fit, maybe I could add another test that only pushes a built model instead.

Reply:
that would be a good way to prove it! 😄